### ISSUE TITLE
Menu in dark theme fails contrast check and is hard to read

### ISSUE THREAD

**Issue by jonmaskrey** [Microsoft/monaco-editor#428]
The contextual menu in the dark themes is hard to read and fails colour contrast for accessibility.

![menu](https://cloud.githubusercontent.com/assets/6881664/25442323/a634648e-2a9c-11e7-86ae-c1c213def872.png)
![contrast-results](https://cloud.githubusercontent.com/assets/6881664/25442324/a6357612-2a9c-11e7-91f4-eb640e9ae7aa.png)

**Comment by chrisdias** [comment]
Should be using the High Contrast theme

**Comment by alexandrudima** [comment]
Let's track the contrast issue for both themes in #428
